docs/usage/20-provisioning-ingresses.adoc

```diff
@@ -51,69 +51,102 @@ In case an invalid `Service` resource is specified as a backend for a given `Ing
 Whenever `dklb` gets to be used as a backend, a Kubernetes event will be emitted and associated with the `Ingress` resource being processed.
 This event contains useful information about the reason why `dklb` is being used instead of the intended backend, and may be used for diagnosing problems.
 
+=== Customizing the target EdgeLB pool
+
+As mentioned before, `dklb` uses sane defaults when provisioning EdgeLB pools for `Ingress` resources annotated for provisioning with EdgeLB.
+It is, however, possible to customize the target EdgeLB pool for a given `Ingress` resource by using the `kubernetes.dcos.io/dklb-config` annotation.
+This annotation accepts a YAML object (henceforth called the _configuration object_) with the following structure:
+
+[source,yaml]
+----
+kubernetes.dcos.io/dklb-config: |  # NOTE: The "|" character is mandatory.
+  name: dklb-pool-0
+  role: "*"
+  network: "dcos"
+  size: 3
+  cpus: 0.2
+  memory: 512
+  frontends:
+    http:
+      port: 8080
+----
+
+The subsections below provide more insight on each of the fields on the configuration object.
+
+[NOTE]
+====
+The `kubernetes.dcos.io/dklb-config` is **OPTIONAL**.
+Whenever said annotation is absent, `dklb` will generate a sane default configuration for the target EdgeLB pool and automatically inject it on the `Ingress` resource.
+====
+
+WARNING: The `kubernetes.dcos.io/dklb-config` cannot be removed after the `Ingress` resource is created.
+
 === Customizing the name of the EdgeLB pool
 
-By default, `dklb` uses the MKE cluster's name and the `Ingress` resource's namespace and name to compute the name of the target EdgeLB pool.
-To specify a custom name for said EdgeLB pool, one may annotate the `Ingress` resource with the following annotation:
+By default, `dklb` uses the MKE cluster's name and a randomly-generated five-digit suffix to compute the name of the target EdgeLB pool.
```

Comment:
Does `dklb` guarantee there are no collisions?

Reply:
It doesn't, but please see my answer above.

Reply:
OK. Please, see my response above too.

Reply:
Addressed in #24